astropy/visualization/lupton_rgb.py

```diff
@@ -47,60 +53,159 @@ def compute_intensity(image_r, image_g=None, image_b=None):
     return np.asarray(intensity, dtype=image_r.dtype)
 
 
-class Mapping:
+class AsinhLuptonStretch(BaseStretch):
+    r"""
+    A mapping for an asinh stretch, with some changes to the constants
+    relative to `~astropy.visualization.AsinhStretch`.
+
+    The stretch is given by:
+
+    .. math::
+        y = \frac{{\rm asinh}(Q x / stretch)}{frac/{\rm asinh}(frac Q)}.
```

Comment:
This hasn't rendered correctly in the docs.

Reply:
Fixed!